dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.36/V2_36_4__add_template_snapshot_to_program_instance_and_remove_fk.sql

```diff
@@ -0,0 +1,6 @@
+alter table programnotificationinstance
```

Comment:
Remember to make all these statements idempotent ("if exists")

Reply:
Yes, good catch!